oak_functions/loader/src/lookup.rs

```diff
@@ -77,43 +64,35 @@ pub fn storage_get_item(
         .get_memory()
         .get(key_ptr, key_len as usize)
         .map_err(|err| {
-            extension.logger.log_sensitive(
-                Level::Error,
-                &format!(
-                    "storage_get_item(): Unable to read key from guest memory: {:?}",
-                    err
-                ),
-            );
+            extension.log_error(&format!(
+                "storage_get_item(): Unable to read key from guest memory: {:?}",
+                err
+            ));
             OakStatus::ErrInvalidArgs
         })?;
-    extension.logger.log_sensitive(
-        Level::Debug,
-        &format!("storage_get_item(): key: {}", format_bytes(&key)),
-    );
-    match extension.lookup_data.get(&key) {
+    extension.log_debug(&format!("storage_get_item(): key: {}", format_bytes(&key)));
+    match extension.get(&key) {
         Some(value) => {
             // Truncate value for logging.
             let value_to_log = value.clone().into_iter().take(512).collect::<Vec<_>>();
-            extension.logger.log_sensitive(
-                Level::Debug,
-                &format!("storage_get_item(): value: {}", format_bytes(&value_to_log)),
-            );
+            extension.log_debug(&format!(
+                "storage_get_item(): value: {}",
+                format_bytes(&value_to_log)
+            ));
             let dest_ptr = wasm_state.alloc(value.len() as u32);
             wasm_state.write_buffer_to_wasm_memory(&value, dest_ptr)?;
             wasm_state.write_u32_to_wasm_memory(dest_ptr, value_ptr_ptr)?;
             wasm_state.write_u32_to_wasm_memory(value.len() as u32, value_len_ptr)?;
             Ok(())
         }
         None => {
-            extension
-                .logger
-                .log_sensitive(Level::Debug, "storage_get_item(): value not found");
+            extension.log_debug("storage_get_item(): value not found");
             Err(OakStatus::ErrStorageItemNotFound)
         }
     }
 }
 
-impl OakApiNativeExtension for LookupExtension {
+impl OakApiNativeExtension for LookupData<Logger> {
```

Comment:
This should be implemented for any Logger, right? (not that it matters for us, but it seems too specific at the moment)

Reply:
Done.